test/cljs/chronograph/routing_test.cljs

```diff
@@ -0,0 +1,23 @@
+(ns chronograph.routing-test
+  (:require [cljs.test :refer-macros [deftest is testing run-tests]]
+            [day8.re-frame.test :as rf-test]
+            [re-frame.core :as rf]
+            [chronograph-web.events.routing :as routing-events]
+            [chronograph-web.subscriptions :as subs]))
+
+(deftest set-token-test
+  (testing "when history token is set"
+    (rf-test/run-test-sync
+     (let [actual (atom nil)]
+       (rf/reg-fx :history-token #(reset! actual %))
+       (rf/dispatch-sync [::routing-events/set-token "/test"])
+       (is (= @actual "/test")
+           "the history-token effect is effected")))))
```

Comment:
Shouldn't this also set the current page?

Reply:
We can't test this because we'd have to depend on window existing. Leaving this for later.